modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -108,6 +123,7 @@
 
 			this.bind( 'onplay', function () {
 				_this.loadAdditionalAssets();
+				_this.syncByReachedCuePoints();
```

Comment:
we would like to invoke explicit sync when user start playing or perform a seek (I verified with @amirch1 that 'onplay' event will be triggered if seeking while playing)